msmbuilder/featurizer/featurizer.py

```diff
@@ -200,6 +200,64 @@ def partial_transform(self, traj):
         return x
 
 
+
+class StrucRMSDFeaturizer(Featurizer):
+    """Featurizer which inputs a trajectory to be analyzed ('traj') and a
```

Comment:
First line of docstrings should be a one sentence summary

More details can go in a paragraph below